examples/nimbus/src/nimbus.idl

```diff
@@ -0,0 +1,45 @@
+namespace nimbus {};
+dictionary ApplicationContext {
+    string? app_id;
+    string? app_version;
+    string? locale;
+    string? device_manufacturer;
+    string? device_model;
+    string? region;
+    string? debug_tag;
```

Comment:
Nice. Does this mean we can deprioritize #188 now?

Reply:
Yes! it makes more sense to me to be nullable!